### DISCUSSION TITLE
Error XA5207 : Could not find android.jar for API level 33

### DISCUSSION THREAD

**Discussion by pburrows** [dotnet/maui#18373] [Q&A]
I just installed Maui and created a new project. It tells me this error when I build:

```
    >Xamarin.Android.Tooling.targets(100,5): Error XA5207 : Could not find android.jar for API level 33. This means the Android SDK platform for API level 33 is not installed. Either install it in the Android SDK Manager (Tools > Android > Android SDK Manager...), or change the Xamarin.Android project to target an API version that is installed. 
```

However, there is no API level 33 available in the SDK manager. Only 34, which I have installed.

![image](https://github.com/dotnet/maui/assets/589029/fc8d0ec8-e532-47c7-945c-eed2cf98f132)

I have searched everywhere in the project for the number "33" and changed it to "34" where it was referencing the Android SDK, but that does not seem to help.

How can I:
A: get Maui to work with API level 34 instead of 33? or,
B: Install API level 33?

PS: I am just trying to create a desktop application, I have zero desire to create an Android application at all!

**Comment by harishprogrammer** [comment]
I'm having same issue, @pburrows have you solved this?
